goals.md

```diff
@@ -0,0 +1,61 @@
+# Goals
+To guide the design of the specification, here we have articulated the high-level goals of this protocol.
+
+## Tying diverse solutions together
+First and foremost, we want to enable various, existing solutions in the lab automation niche to interoperate, irrespective of programming language, application niche (GUI, driver, orchestration, data storage,...) or project origin.
+
+In the past, we have [observed](https://github.com/pymeasure/python-lab-automation-landscape) that the lab automation landscape, at least in Python, is quite fragmented.
+As it has turned out unrealistic to unify/agglomerate these efforts into larger, and thus more resilient, projects, we have changed tack.
+We want to enable users to pick drivers from some instrument control libraries with a reasonably homogeneous/simple API, a nice GUI from another project, and a data storage solution from elsewhere, and tie them together to serve the needs of _their_ setup.
+
+[PyMeasure](https://pymeasure.readthedocs.io) is our project, so we know it best, and this will surely influence the design effort.
+Also, we plan a reference implementation in Python that will also probably be primarily tested with PyMeasure.
+However, we intend PyMeasure to be only one of _many_ "customer" libraries that can interface with ECP.
+
+This specification will be independent of the programming language its Components are written in, so it will be possible to mix-and-match.
+Instead of implementing all the necessary Components, we want to give the communication tools to connect everything, to balance on the shoulders of giants.
+
+## Small to medium-sized use cases
+We want ECP to serve laboratory setups of up to approximately 50 participants on one or more computation Nodes on a common network.
+For example, you might have up to ~dozens of instruments, a few data acquisition nodes, one or several GUIs and control nodes.
```

Comment:
Do you mean node in the sense of #19 ?
I got confused, what you understand under "Node".

Is a "computation Node" a physical computer? But in line 20 you have "data acquistion [...] nodes" - do you mean Components?

Would that still fit your idea:
```suggestion
We want ECP to serve laboratory setups of up to approximately 50 participating Components on one or more computation nodes (e.g. personal computers) on a common network.
For example, you might have up to ~dozens of instruments, a few data acquisition Components, one or several GUIs and control Components.
```

Reply:
Ah, that was left over from copy-pasting, thanks for the correction. I meant "Components" here.

I'll integrate your suggestions locally.